code/datums/supply_packs/clothing.dm

```diff
@@ -1,110 +1,7 @@
-//---------------------------------------------
-//POUCHES BELOW THIS LINE
-//---------------------------------------------
-/datum/supply_packs/pouches_general
-	name = "general pouches crate (2x normal, 1x medium, 1x large)"
-	contains = list(
-					/obj/item/storage/pouch/general,
-					/obj/item/storage/pouch/general,
-					/obj/item/storage/pouch/general/medium,
-					/obj/item/storage/pouch/general/large
-					)
-	cost = RO_PRICE_VERY_CHEAP
-	containertype = /obj/structure/closet/crate
-	containername = "general pouches crate"
-	group = "Clothing"
-
-/datum/supply_packs/pouches_weapons
-	name = "weapons pouches crate (1x bayonet, pistol, explosive)"
-	contains = list(
-					/obj/item/storage/pouch/bayonet,
-					/obj/item/storage/pouch/pistol,
-					/obj/item/storage/pouch/explosive
-					)
-	cost = RO_PRICE_VERY_CHEAP
-	containertype = /obj/structure/closet/crate
-	containername = "weapons pouches crate"
-	group = "Clothing"
-
-/datum/supply_packs/pouches_ammo
```

Comment:
should keep this, large magazine pouches are limited

Reply:
ok created a new crate with the large mag pouche

Reply:
done

Reply:
it's fine for me as long as your comment are productive... i corrected it.